src/loinclib/loinc_tree_schema.py

```diff
@@ -1,10 +1,19 @@
+"""Schema for LOINC hierarchy"""
 from enum import StrEnum
 
+class LoincEdges(StrEnum):
+    """Superclass for all edges from LOINC"""
+    pass
 
-class LoincTreeEdges(StrEnum):
+class LoincTreeEdges(LoincEdges):
+    """Edges from the LOINC tree browser."""
     tree_parent = "tree_parent"
 
+class LoincDanglingNlpEdges(LoincEdges):
```

Comment:
The new edge that @ShahimEssaid requested. I also created a new, shared superclass `LoincEdges` which is the parent of both `LoincDanglingNlpEdges` and `LoincTreeEdges`, though there is no logic that uses it.